components/admin_console/index.ts

```diff
@@ -29,6 +26,11 @@ import LocalStorageStore from 'stores/local_storage_store';
 
 import {GlobalState} from 'types/store';
 
+import {selectStaticItem} from 'actions/lhs';
+
+import {Role} from '@mattermost/types/roles';
```

Comment:
For some reason, those lines triggered this eslint rule in my editor – reverted that.
```
`@mattermost/types/roles` import should occur after import of `types/store` (import/order)
```

Reply:
That's probably the annoying error that happens when `@mattermost/types` hasn't been compiled. It shows up as an import order error rather than a missing file error for whatever reason

How do you generally run the web app? When either `make run` or `make dev` is running, it should have been recompiled automatically

Reply:
I tried the `make dev` and I think is the most proper way to go in general, but `make run` is more solid for me.

Reply:
Yeah, `make run` is the one I use personally, so I guarantee that it keeps working. 😛

We try to ensure `make dev` works as well since some people like the hot reloading features, but since most people don't use it, it sometimes ends up breaking